types/stylelint/index.d.ts

```diff
@@ -529,16 +529,6 @@ declare module 'stylelint' {
 			_specifiedConfigCache: Map<Config, Promise<CosmiconfigResult>>;
 			_postcssResultCache: Map<string, PostCSS.Result>;
 			_fileCache: FileCache;
-
-			_getPostcssResult: (options?: GetPostcssOptions) => Promise<PostCSS.Result>;
-			_lintSource: (options: GetLintSourceOptions) => Promise<PostcssResult>;
-			_createStylelintResult: (
-				postcssResult: PostcssResult,
-				filePath?: string,
-			) => Promise<LintResult>;
-
-			getConfigForFile: (searchPath?: string, filePath?: string) => Promise<CosmiconfigResult>;
-			isPathIgnored: (s?: string) => Promise<boolean>;
```

Comment:
[note] I believe `InternalApi` and `createLinter` is private and never break our ecosystem.

https://github.com/stylelint/stylelint/blob/d40288983161124d7b8a29be219835bf7ea6841d/types/stylelint/index.d.ts#L458-L463

However, if this pull request would harm the ecosystem, I'd like to close it.